src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -234,11 +234,11 @@
   <data name="GenerateLocal" xml:space="preserve">
     <value>Generate local '{0}'</value>
   </data>
-  <data name="GenerateForInNewFile" xml:space="preserve">
-    <value>Generate {0} for '{1}' in '{2}' (in new file)</value>
+  <data name="Generate_0_1_in_new_file" xml:space="preserve">
```

Comment:
I far prefer this form.  Now we dont' have this bizarre inconsistency between the identifier and the text it is replaced with.  There's a standard pattern for how you get the identifier given the format string.

Over in TS we just generated these identifiers automatically.  Thus, all you ever had to do was care about the format string.  Seems much nicer.   And much easier to grok in code.  "Oh... this needs X many arguments.  And here's where each goes.  And here's what the string will actally be"  All without needing to hover or anything.